index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Task list</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Task List</h1>
+
+      <form id="new-task-form">
+        <input type="text" id="new-task-input" placeholder="Enter Data..." />
+        <input type="submit" id="new-task-submit" value="Add task" />
+
+      
+    </header>
+    <div id="message">
+  
+</div>
+  </form>
```

Comment:
<form> is inside <header> but closing </form> closing tag is outside of header. Please fix this.